guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -37,12 +44,27 @@ On CentOS, configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/le
 # curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
 ----
 endif::[]
+
 +
-On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
+.. On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 +
 ----
 # subscription-manager repos --enable=rhel-7-server-extras-rpms
 ----
+.. Enable the `rhel-7-server-rpms` repository:
++
+----
+# subscription-manager repos --enable=rhel-7-server-rpms
+----
+.. Enable the `rhel-7-server-satellite-{ProjectVersion}-rpm` repository:
++
+[options="nowrap" subs="quotes, attributes"]
+----
+# subscription-manager repos --enable=rhel-7-server-satellite-{ProjectVersion}-rpm
+----
```

Comment:
I am torn on these. For connected setups, we don't call them out, as we expect them to be present anyways.
And in disconnected, we only *explicitly* say what to do in the "install satellite" and "upgrade satellite to a new version" documents, any other place that involved installing packages is silent about the fact that repos need to be available, as I think we expect people not to *remove* them?

@ehelms do you know what the "policy" here is? I'd love to avoid additional text/steps at all costs ;)

Reply:
@evgeni @ehelms How should I proceed with this?

Reply:
If these should already be enabled at this point in the lifecycle of the Satellite itself, having been an installed product that is actively running I think we can skip the need to call these out.

Reply:
Removed.